packages/apm/src/integrations/tracing.ts

```diff
@@ -202,6 +213,16 @@ export class Tracing implements Integration {
       });
     }
 
+    // tslint:disable-next-line: no-non-null-assertion
+    if (this._options!.discardBackgroundSpans && global.document) {
```

Comment:
Why do we have `this._options!` in the first place? We should allow `this._options` to be undefined by this point.